src/features/persons/app_user_profile/index.tsx

```
@@ -0,0 +1,60 @@
+          gap: '12px',
+          display: 'flex',
+          flexDirection: 'column',
+        }}
+      >
+        <Typography className="h2" color="var(--black)">
+          {t('tr_appUserProfile')}
+        </Typography>
+        <Typography
+          className="body-regular"
+          color="var(--black)"
+          dangerouslySetInnerHTML={{ __html: getTextForAppPersonProfileDesc() }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)